### PR TITLE
Change cursor to pointer on note remove button

### PR DESCRIPTION
**Pull request by helloRupa**
Added style for glyphicon-remove.

### CODE REVIEW

**Comment by 1995YogeshSharma** [comment]
Please move this to new css file created. That would resolve the merge conflict as well.

File: stopWatch.html [hunk 1]
```diff
@@ -273,7 +273,10 @@
     		li .list-item {
     			overflow: hidden;
     			text-overflow: ellipsis;
-    		}
+        }
```
Comment (1995YogeshSharma) [inline stopWatch.html]: merge it with master locally and move this style to css file.

**Comment by helloRupa** [comment]
I believe the conflicts are resolved now. I tested the code again and it still appears to be working.

**Comment by 1995YogeshSharma** [comment]
Thanks!
Merging the PR

**Pull request merged**